### ISSUE TITLE
The name 'ImageDecoder' does not exist in the current context

### ISSUE THREAD

**Issue by michaellu88** [mochi-neko/StbImageSharpForUnity#1]
i have copy the dependency into minifest. but i have the ImageDecoder does not exist in the current context.
pls help
tq

**Comment by michaellu88** [comment]
i have solve the problem, not sure is that a correct way, i just remove all the .asmdef file. Then i can access ImageDecoder.

**Comment by mochi-neko** [comment]
@michaellu88 

I released the version 1.1.2 that can refer the `ImageDecoder` from the Assembly-CSharp (the default assembly create by the Unity).

Please update your manifest by specifying version #1.1.2 as follows:

```
dependencies: {
    "stbsharp.stbimagesharp": "https://github.com/mochi-neko/StbImageSharpForUnity.git?path=/Assets/StbImageSharp#1.1.2",
    "mochineko.stbimagesharp-for-utity": "https://github.com/mochi-neko/StbImageSharpForUnity.git?path=/Assets/Mochineko/StbImageSharpForUnity#1.1.2",
}
``` 
.

Also you can refer the `ImageDecoder` by creating an Assembly Definition Asset on your project and adding the `StbSharp.StbImageSharp` & the `Mochineko.StbImageSharpForUnity` to the `Assembly Definition References`.

See also [this manual](https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html).

**Comment by michaellu88** [comment]
Tq for your prompt reply.
